src/commands/changeinvite.js

```diff
@@ -65,6 +65,8 @@ class ChangeInviteCommand extends Command {
                     name: invite?.guild.name ?? guild.name,
                 });
 
+                client.emit('updateGuilds')
```

Comment:
explicacao no primeiro comentario em src/events/updateGuilds.js
```suggestion
                client.emit('updateGuilds', false);
```